### DISCUSSION TITLE
Checkbox label with link

### DISCUSSION THREAD

**Discussion by vicainelli** [mantinedev/mantine#905] [Q&A] [answered]
Hi all!

Does anyone know if it's possible to include a link inside the `label` props on the Checkbox component?

Like this

```html
<label>
  <input type="checkbox" />
  I agree with the <a href="#">terms</a>
</label>
```

**Comment by rtivital** [comment, marked as answer]
https://codesandbox.io/s/autumn-moon-tjz4ds?file=/src/App.tsx

**Reply by vicainelli** [reply]
Awesome! 
Thanks @rtivital
